pyproject.toml

```diff
@@ -45,6 +45,9 @@ parameterized = "^0.8.1"
 nautobot = ["nautobot"]
 nautobot-device-lifecycle-mgmt = ["nautobot-device-lifecycle-mgmt"]
 
+[tool.poetry.group.dev.dependencies]
+nautobot-device-lifecycle-mgmt = {version = "^1.0.2", optional = true}
```

Comment:
In this scenario you may want to remove the `optional = true` because in development it should be required.

Reply:
Updated! Thanks @bryanculver.